libs/netpgp/meson.build

```diff
@@ -2,14 +2,70 @@ project('netpgp', 'c',
         version: '20140220',
         license: '2-part BSD')
 
-# pkg-config dependencies
+cc = meson.get_compiler('c')
+
+
+## dependencies
 pthreads = dependency('threads')
 zlib = dependency('zlib', fallback: ['zlib', 'zlib_dep'])
 openssl = dependency('openssl', fallback: ['openssl', 'dep'])
 
-# Dependencies without pkg-config, just try linking against them
-cc = meson.get_compiler('c')
+# BZip2 has no pkg-config, just try linking against it
+bzip2 = cc.find_library('bz2', required: get_option('bzip2'))
```

Comment:
why do we still link against bzip2 here?

Reply:
huh, apparently email replies don't thread terribly well.  https://github.com/deltachat/deltachat-core/pull/310#issuecomment-424704208 was meant to be a response to this